src/l1-resources/online-archive/index.ts

```diff
@@ -126,9 +134,9 @@ export function toJson_CfnOnlineArchiveProps(obj: CfnOnlineArchiveProps | undefi
  */
 export enum CfnOnlineArchivePropsCollectionType {
   /** STANDARD */
-  STANDARD = 'STANDARD',
+  STANDARD = "STANDARD",
```

Comment:
can you create a ticket to enable prettier for this repo so this things are consistent?

Reply:
sure. I create this ticket https://jira.mongodb.org/browse/INTMDB-746